net-tools/netwatch/src/netmon/windows.rs

```diff
@@ -119,9 +122,12 @@ impl CallbackHandler {
         handle: UnicastCallbackHandle,
     ) -> Result<()> {
         trace!("unregistering unicast callback");
-        if self.unicast_callbacks.remove(&handle.0 .0).is_some() {
+        if self.unicast_callbacks.remove(&(handle.0 .0 as isize)).is_some() {
```

Comment:
I did a lot of casting here to get things to align and tests pass, but this is scary and I'm not a 100% confident in these.

Reply:
it's fine, honestly I just use `as _` in this case, as these numbers depend on the lib giving you the right type anyway